visual-js/visual/src/cli/baselines.ts

```diff
@@ -23,7 +23,8 @@ export async function baselineMergeCommand(_: unknown, cmd: Command) {
   const visualApi = apiFromOptions(options);
   const result = await visualApi.mergeBaselines({
     projectName: options.project,
-    ...options,
+    sourceBranch: options.sourceBranch,
+    targetBranch: options.targetBranch,
```

Comment:
Should we check that they are different ? (and that this is non null).

That would permit to produce cleaner errors

Reply:
1. The null check should be carried out by commander, since those are required options.
2. Target and source branch can be the same. In this case nothing is copied (since `created_at(target) < created_at(source)` is false for all baselines)